osu.Game/Overlays/Profile/Sections/ProfileItemContainer.cs

```diff
@@ -16,12 +16,19 @@ public class ProfileItemContainer : Container
 
         protected override Container<Drawable> Content => content;
 
-        private Color4 idleColour;
-        private Color4 hoverColour;
-
         private readonly Box background;
         private readonly Container content;
 
+        private Color4 idleColour;
+
+        public Color4 IdleColour
+        {
+            get => idleColour;
+            set => idleColour = background.Colour = value;
+        }
+
+        public Color4 HoverColour { get; set; }
```

Comment:
I'm not sure I like these being public too much, the component is kind of tightly coupled to the theme so just being able to set any colour feels off. In general `DrawableMostPlayedBeatmap` is awkwardly written imo, as the `ProfileItemContainer` should be the top-level component there (or even, it should inherit from it). I'd envision something like the following:

```
DrawableMostPlayedBeatmap : ProfileItemContainer
    GridContainer
        Column (GridSizeMode.Absolute, width = cover_width)
            UpdateableBeatmapSetCover
        Column (GridSizeMode.Distributed)
            FillFlowContainer
                MostPlayedBeatmapMetadataContainer
                LinkFlowContainer
        Column (GridSizeMode.AutoSize)
            PlayCountText
```

Then you can make these `protected`. I'm aware this is a little bit out there in terms of work required but I believe it'll lead to cleaner code.

Reply:
Solved this in a bit simpler way - added a `MostPlayedBeatmapContainer` deriving from the `ProfileItemContainer` (replacing its existing usage here) and set its colours in its own BDL, allowing mentioned fields to be set to protected. 

I know the whole component is still in an awkward state, but I'd wanna avoid rewriting it if possible (for now).